rllib/connectors/tests/test_agent.py

```diff
@@ -394,14 +395,23 @@ def test_vr_connector_with_multiple_buffers(self):
         # without reward-to-go.
         view_rq_dict = {
             # obs[t-context_len+1:t]
-            "context_obs": ViewRequirement("obs", shift=f"-{context_len-1}:0"),
+            "context_obs": ViewRequirement(
```

Comment:
These view requirements where missing a proper space to be checked in agent collector (default space has `.shape=()`